fs/operations/operations.go

```diff
@@ -556,7 +556,7 @@ func Move(ctx context.Context, fdst fs.Fs, dst fs.Object, remote string, src fs.
 		newDst, err = doMove(ctx, src, remote)
 		switch err {
 		case nil:
-			fs.Infof(src, "Moved (server side)")
+			fs.Infof(src, "Moved (server side) to %s", newDst.String())
```

Comment:
This should also only print if the destination string is different as most of the time it will be the same.